### DISCUSSION TITLE
Sobre média salarial

### DISCUSSION THREAD

**Discussion by dan-liberato** [frontendbr/forum#1464] [Discussão]
Qual seria a média salarial de um Dev que faz (Front-end e Back-end):

Sites
Temas para WordPress
Aplicativos (React Native)
Sistemas/Softwares em PHP e as vezes algo em JS e Node
Design UI/UX
Landing Pages
Cuida de toda a Infra
Manutenção e migração (Quando necessário)
Gerência de tudo, inclusive datas de pagamento e compra de domínios

Existe uma média salarial para isso?

**Comment by willycamargo** [comment]
De **R$1.000** á **R$25.000**. 
Não existe média confiável pra isso.

**Comment by felipefialho** [comment]
Isso depende de:

- Tempo de mercado
- Qualidade técnica
- Região
- Tamanho da empresa
- Funções dentro da empresa
- Valor agregado
- Etc

São muitas variáveis que definem, de fato não existe uma média.

Mas [pode ter uma idéia aqui](https://survey.frontendbr.com.br).

**Comment by salatielsql** [comment]
@dan-liberato parece que você é o dono da empresa! uahsuahusah

**Comment by dan-liberato** [comment]
Obrigado por responderem e @salatielq, parece mesmo ahuauhuah

Vai fazer 5 anos que entrei nessa área e como sou do interior de SP, trabalhei praticamente 4 nessa empresa, não tinha paz, era de domingo a domingo. Com o tempo, chegou no meu limite e desanimei, pedi demissão, até pensei em mudar de área, mas gosto dessa.

Agora é criar um portfólio, atualizar LinkedIn e correr atrás de novas oportunidades.

Desculpe o desabafo!

**Comment by rubenmarcus** [comment]
Nunca vi front tirar mais que 15mil no Brasil a não ser que faça freelas muito bons, ou que trabalhe pra fora do pais. Em emprego no mercado, o teto pelo menos nas vagas que vejo é 15mil.

a média acho que pra junior é de 1k- 3k, pleno de 3k - 7k, senior de 7k - 15k, dependendo dos fatores ali que o felipe falou. Isso em SP.
